virtool/api/response.py

```diff
@@ -88,13 +88,20 @@ async def error_middleware(req: web.Request, handler: Callable):
     :param req: the incoming request
     :param handler: the next middleware to be executed
     """
+    logger = logging.getLogger()
     try:
+        logger.error("BEFORE HANDLE")
         return await handler(req)
 
     except web.HTTPException as exc:
+        logger.error("CAUGHT ERROR")
```

Comment:
* Don't leave these dev logs in your PR.
* We always instantiate `logger` at the top of the module.